revolution/motor.py

```diff
@@ -124,3 +128,46 @@ def _revolution(self) -> None:
             with self.environment.contexts() as contexts:
                 contexts.motor_revolution_period = revolution_period
                 contexts.motor_speed = motor_speed
+
+    def cruise_control(self) -> None:
+        with self.environment.contexts() as contexts:
+
+            # values from from cruise_control_pi in GEN11_MCMB
+            k_p = 250.0
+            k_i = 0.015
+            k_d = 0.0
+            
+            integralMin = -200.0
+            integralMax = 200.0
+            integrator = 0.0
+
+            derivativeMin = -100.0
+            derivativeMax = 100.0
+            derivative = 0.0
+
+            outMin = -255.0
+            outMax = 255.0
+            output = 0.0
+
+            timeStep_ms = 20
+            error = 0.0
+            prevError = 0.0
```

Comment:
Perhaps this should be exported to the settings. But, I fear that the settings are already too bloated. We will keep it as is for now.